web/themes/custom/novel/templates/views/views-view-unformatted--list-page.html.twig

```diff
@@ -0,0 +1,7 @@
+<ul class="event-list">
+  {% for row in rows %}
+    <li class="event-list__item{{ row.isStacked ? ' event-list__item--stacked' }}">
```

Comment:
The specific `event-list` class pops up in a generic `list-page` again here.

Also: Should this be in `/components`?  It seems to be the pattern for reusable templates.